plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -103,13 +104,7 @@
 import io.trino.spi.statistics.ComputedStatistics;
 import io.trino.spi.statistics.TableStatistics;
 import io.trino.spi.statistics.TableStatisticsMetadata;
-import io.trino.spi.type.LongTimestamp;
-import io.trino.spi.type.LongTimestampWithTimeZone;
-import io.trino.spi.type.TimeType;
-import io.trino.spi.type.TimestampType;
-import io.trino.spi.type.TimestampWithTimeZoneType;
-import io.trino.spi.type.TypeManager;
-import io.trino.spi.type.VarcharType;
+import io.trino.spi.type.*;
```

Comment:
<img width="1005" alt="Screenshot 2023-12-02 at 19 54 44" src="https://github.com/trinodb/trino/assets/66972/88e6aaae-3d22-4534-9a4b-4e2465f90f45">

Reply:
I think it was mentioned here - https://trino.io/development/#code-style - We need to import this [configuration](https://github.com/airlift/codestyle) to IntellIJ so that it could apply the changes at its end. Let me know if you face any issues in it

Reply:
Aha, I didn't know you have to import this. I thought it's picked up automatically by IntelliJ